google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
> IMO there's no better way to convey that no callback is being passed than using a no-arg value.
👍 

WRGT to passing Optional as a parameter I am strongly against it. It's very ungainly from a callers point of view and I don't see a lot of benefits.  Generally optional/nullable parameter types are like flag parameter types. An anti-pattern that should be avoided.